src/wp-includes/block-supports/block-style-variations.php

```diff
@@ -19,6 +21,7 @@
  * @return string The unique variation name.
  */
 function wp_create_block_style_variation_instance_name( $block, $variation ) {
+	_deprecated_function( __FUNCTION__, '6.7.0', "wp_unique_id( \$variation . '--' )" );
```

Comment:
Is this a valid illustration of this deprecated function's replacement? Or, should it just be `'wp_unique_id'`?

Reply:
It should be just `'wp_unique_id'`.
```
_deprecated_function( __FUNCTION__, '6.7.0', 'wp_unique_id' );
```

I checked the rest of the codebase for usage of parameters in the call to `_deprecated_function`. It's always just the function name.

Reply:
Thanks, I'll get this updated.